docs/components/console/manage-clusters/settings.md

```diff
@@ -1,13 +1,50 @@
 ---
 id: settings
 title: Settings
-description: "Set up resource-based authorizations to authorize access and assign permissions to process and decision definitions."
+description: "Manage your cluster settings or permanently delete the cluster."
 ---
 
-Set up resource-based authorizations to authorize access and assign permissions to process and decision definitions in the **Settings** tab. You can enable resource-based authorizations on a per-cluster basis. It's possible to disable the feature at any time.
+In this document, learn how to manage your cluster settings using resource-based authorizations, automatic cluster updates, and user task restrictions, or permanently delete the cluster.
```

Comment:
Gave a tiny bit more of a description here.

Reply:
Thanks @christinaausley I took out "In this document, learn how to" as this seemed superfluous, but kept in the detail about the individual settings, I had removed this originally to really make it concise (i.e. not detail every setting in the intro) but I think you're right, it helps to have it there.